app/static/css/style.css

```diff
@@ -94,3 +95,9 @@ img {
 *[cursor="cell"] > * {
   cursor: cell !important;
 }
+
+#remote-screen[fullscreen="true"] > img {
+  object-fit: contain;
+  height: 100%;
+  width: 100%;
+}
```

Comment:
This ends up breaking the mouse integration. The image needs margins otherwise it extends to the ends of the screen and the coordinates for mouse events get screwed up. The fix is to borrow CSS from #196:

```css

#remote-screen img {
  max-width: 100%;
}

#remote-screen[fullscreen="true"] > img {
  margin: auto;
  height: 100%;
  max-height: 100vh;
  max-width: 100vw;
}
```

Can we move this up to line 56 so all the `#remote-screen` rules are together?